ee-feature-pack/galleon-common/src/main/resources/feature_groups/domain-host-excludes.xml

```diff
@@ -7,35 +7,65 @@
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.health&quot;,&quot;org.wildfly.extension.metrics&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
     </feature>
     <feature spec="domain.host-exclude">
         <param name="host-exclude" value="WildFly13.0"/>
         <param name="host-release" value="WildFly13.0"/>
-        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.health&quot;,&quot;org.wildfly.extension.metrics&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
     </feature>
     <feature spec="domain.host-exclude">
         <param name="host-exclude" value="WildFly14.0"/>
         <param name="host-release" value="WildFly14.0"/>
-        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.health&quot;,&quot;org.wildfly.extension.metrics&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
     </feature>
     <feature spec="domain.host-exclude">
         <param name="host-exclude" value="WildFly15.0"/>
         <param name="host-release" value="WildFly15.0"/>
-        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;]"/>
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.health&quot;,&quot;org.wildfly.extension.metrics&quot;,&quot;org.wildfly.extension.health&quot;]"/>
```

Comment:
The health extension is included twice here.

Reply:
I really want it to be excluded! :)
sorry about that

Reply:
fixed